client/landing/stepper/declarative-flow/internals/steps-repository/domains/index.tsx

```diff
@@ -297,7 +299,7 @@ const DomainsStep: Step = function DomainsStep( { navigation, flow } ) {
 				<FormattedHeader
 					id="domains-header"
 					align={ flow === HUNDRED_YEAR_PLAN_FLOW ? 'center' : 'left' }
-					subHeaderAlign={ flow === HUNDRED_YEAR_PLAN_FLOW ? 'center' : 'left' }
+					subHeaderAlign={ flow === HUNDRED_YEAR_PLAN_FLOW ? 'center' : null }
```

Comment:
This is a fix since `FormattedHeader` subHeaderAlign only accepts 'center' or null.

Reply:
Nope, align [accepts](https://github.com/Automattic/wp-calypso/blob/trunk/client/components/formatted-header/index.jsx#L77) 'center', 'left' and 'right'